tutorial/tutorial-template.md

```diff
@@ -0,0 +1,75 @@
+# Title
+
+{Before using this template, read the accompanying [tutorial template guide](tutorial-template-guide)}.
+
+## Overview
+
+In this tutorial, you'll learn how to {top learning objective} using {product}. This tutorial is intended for {audience}. It assumes you have basic knowledge of:
```

Comment:
Is there another word I can use instead of "top learning objective?" Main learning objective?

Like the "main" or "top" learning objective could be _create components in React_. And then the learning objectives you list below might include:

* Modify arguments that are passed to React components
* Describe the differences between functional and class components
* ....

Open to any suggestions here.

Reply:
For me, I'd maybe say: "In this tutorial you'll learn how to {insert brief description of the main tutorial task}." I'd leave it at that because your accompanying guide goes into a lot more detail about it and they can just refer to that.

Actually, I think it would be good to check that this section closely matches your guide section. I haven't done a direct comparison, but I feel like there might be subtle differences when I think of it from memory?

Reply:
@barbaricyawps "Brief description of the main tutorial task" might be exactly what I'm looking for. That first line can briefly describe what you're going to do, and later you can list the specific learning objectives users will achieve by the end of the tutorial. 

I can update my tutorial template...it is a tad out of sync. Thank you!